snapcraft/internal/build_providers/_multipass/_multipass.py

```diff
@@ -198,9 +199,12 @@ def _mount(self, host_source: str, target: str) -> None:
             return
 
         target = "{}:{}".format(self.instance_name, target)
-        uid_map = {str(os.getuid()): "0"}
-        gid_map = {str(os.getgid()): "0"}
-
+        if platform.system() != "Windows":
```

Comment:
nit: the rest of the code base is using `if sys.platform == "win32"` for this check, perhaps there's some value in consistency.

Either way, :+1: Thank you @NickZ!

I haven't given Windows attention in a while and need to circle back now...  Are you having success with it? :-)